### ISSUE TITLE
Markdown parsing fails

### ISSUE THREAD

**Issue by greenape** [greenape/mknotebooks#52]
```  
File "/opt/hostedtoolcache/Python/3.8.5/x64/lib/python3.8/site-packages/markdown/extensions/md_in_html.py", line 78, in run
    block = self._process_nests(element, block)
  File "/opt/hostedtoolcache/Python/3.8.5/x64/lib/python3.8/site-packages/markdown/extensions/md_in_html.py", line 45, in _process_nests
    self.run(element, block[nest_index[-1][0]:nest_index[-1][1]],  # last
  File "/opt/hostedtoolcache/Python/3.8.5/x64/lib/python3.8/site-packages/markdown/extensions/md_in_html.py", line 52, in run
    tag = self._tag_data[self.parser.blockprocessors.tag_counter]
IndexError: list index out of range
```

Still have some issues after pulling the changes

_Originally posted by @lanking520 in https://github.com/greenape/mknotebooks/issues/46#issuecomment-694616898_

**Comment by greenape** [comment]
So this is coming up when MkDocs tries to parse the markdown, and is I thinks actually this issue https://github.com/Python-Markdown/markdown/issues/780

So it may be that a bit of jiggling the template around will resolve things.

**Comment by greenape** [comment]
@lanking520 have you got a notebook you can share that triggers it?

**Comment by greenape** [comment]
I _think_ this is going to be fixed once markdown 3.3 is released.

**Comment by lanking520** [comment]
We got the problem fixed once we upgrade to mknotebook 0.6.1. Everything works perfectly:

generated page: http://docs.djl.ai/jupyter/load_pytorch_model.html

**Comment by greenape** [comment]
Great, thanks 👍
